platform/ui/src/components/FilterWrapper/FilterWrapper.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
```

Comment:
Maybe renamed "FormGroup"
Similar semantics of Bootstrap where you wrap label + input + extra
Not necessarily limited to filtering